editor.planx.uk/src/@planx/components/TextInput/Public.tsx

```diff
@@ -60,9 +74,10 @@ const TextInputComponent: React.FC<Props> = (props) => {
             onChange={formik.handleChange}
             errorMessage={formik.errors.text as string}
             id={props.id}
+            textLength={props.type}
             inputProps={{
               "aria-describedby": [
-                props.description ? DESCRIPTION_TEXT : "",
+                props.description ? `${DESCRIPTION_TEXT} character-hint` : "",
```

Comment:
added a hint for screen readers to introduce character count when you first click on it